bidding_auction_services_multi_seller_auctions.md

```diff
@@ -459,92 +459,92 @@ service?
 
 ![server orchestrated comp auction](images/server-orchestrated-comp-auction1.png)
 
-_This type of multi seller auction will be supported for app and web advertising with Bidding and Auction services._ 
+_This type of multi seller auction will be supported for app and web advertising with Bidding and Auction services, including [Protected App Signals](https://github.com/privacysandbox/protected-auction-services-docs/blob/main/bidding_auction_services_protected_app_signals.md#multi-seller-auction-support) data._
 
 #### High-level overview
 The top-level seller code in the publisher web page in the browser would send a [unified request][16] to the seller's
 ad service that includes separate contextual request payload for each seller participating in the auction and one
-encrypted Protected Audience request payload. The top-level seller’s code in the publisher web page asks the browser
-for this encrypted Protected Audience data to include in the unified request.
+encrypted Protected Audience request payload. The top-level seller’s code in the publisher web page/app asks the browser/android API
+for the encrypted Protected Audience/Protected App Signals data to include in the unified request.
```

Comment:
Should we instead do the following here:

`s@Protected Audience/Protected App Signals@Protected Auction(Protected Audience OR Protected App Signals)@` and then use Protected Audience everywhere else below?